### PR TITLE
Rack intel

### PR DESCRIPTION
**Pull request by RackerKate**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-8b7d691da2/how-to/)

File: content/rackspace-intelligence/getting-started-with-rackspace-intelligence-for-the-cloud.md [hunk 1]
```diff
@@ -1,102 +1,83 @@
...
 -   Rackspace cloud databases
--   Other entities, which need not be hosted at Rackspace and need not
-    be servers or databases
-
-Only data available to Rackspace Intelligence can be gathered and acted
-on. For example, if you define a website that is hosted outside
-Rackspace as an entity, then you might be able to usefully place a check
-on its ping time, but you probably won't to be able to check the space
-available in its file system.
+-   Other entities that may be hosted by Rackspace or an another service
```
Comment (stephamon) [inline content/rackspace-intelligence/getting-started-with-rackspace-intelligence-for-the-cloud.md]: ...that might be hosted...

File: content/rackspace-intelligence/getting-started-with-rackspace-intelligence-for-the-cloud.md [hunk 2]
```diff
@@ -1,102 +1,83 @@
...
 
-**Note**: Early publications about Rackspace Intelligence referred to it
-as *Cloud* Intelligence. Because the service is meant to be useful for
-both cloud and dedicated devices, it is now
-named *Rackspace* Intelligence. The Rackspace Intelligence team is working
-to add support for dedicated accounts.
+Rackspace Intelligence can only gather and act on the information that is
```
Comment (stephamon) [inline content/rackspace-intelligence/getting-started-with-rackspace-intelligence-for-the-cloud.md]: Rackspace Intelligence can gather and act on only the information...

File: content/rackspace-intelligence/getting-started-with-rackspace-intelligence-for-the-cloud.md [hunk 3]
```diff
@@ -1,102 +1,83 @@
...
+available to it. For example, if you define a website that is hosted outside of
+Rackspace as an entity, then you might be able to place a check on its ping
+time, but you probably won't to be able to check the space available
```
Comment (stephamon) [inline content/rackspace-intelligence/getting-started-with-rackspace-intelligence-for-the-cloud.md]: ...you probably won't be able to...

File: content/rackspace-intelligence/getting-started-with-rackspace-intelligence-for-the-cloud.md [hunk 4]
```diff
@@ -1,102 +1,83 @@
...
+### Explore the Rackspace Intelligence dashboard
 
-Before you can use [Rackspace
-Intelligence](https://intelligence.rackspace.com/), you must be logged
-in to your [Cloud Control Panel](https://login.rackspace.com/)
-account. For instructions, see [Log in to Rackspace
-Intelligence](/how-to/logging-in-to-the-rackspace-intelligence-dashboard).
+Actions that you can perform in the user interface are grouped into four
+sections: **Monitor**, **Alert**, **Suppress**, and **Visualize**. Within
+each section, you can drill down to view details. For more information, see
```
Comment (stephamon) [inline content/rackspace-intelligence/getting-started-with-rackspace-intelligence-for-the-cloud.md]: "drill down" is kind of jargon-y. Maybe something like "Within each section, you can view details about the action."?

File: content/rackspace-intelligence/getting-started-with-rackspace-intelligence-for-the-cloud.md [hunk 5]
```diff
@@ -125,84 +104,94 @@ of known entities, you can perform management functions such
...
 The [list of open
 alerts](/how-to/monitoring-open-alerts-with-rackspace-intelligence) identifies
 all entities that are currently in a status for which you have asked
-Rackspace Intelligence to trigger alarms. Depending on the [notification
+Rackspace Intelligence to trigger alarms.
+
+Depending on the [notification
 plans](/how-to/working-with-rackspace-intelligence-notification-plans) that
 you have defined, appropriate communications and actions might have
 already occurred in response to these alerts. Examining the list
```
Comment (catlook) [inline content/rackspace-intelligence/getting-started-with-rackspace-intelligence-for-the-cloud.md]: Hmmm how about "Examine the list directly to investigate ...."  ?

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-bdbec5f22f/how-to/). :bow:

**Pull request merged**